src/object.rs

```diff
@@ -1490,6 +1496,52 @@ impl<T: ObjectType> ObjectExt for T {
         }
     }
 
+    fn set_qdata(&self, key: Quark, value: Box<dyn Any>) {
+        extern "C" fn drop_value(ptr: glib_sys::gpointer) {
+            if !ptr.is_null() {
```

Comment:
Why would it ever be `NULL`?

Reply:
Theoretically it wouldn't. Should I remove it?

Reply:
You could add an assertion :)

Reply:
Done